lms/urls.py

```diff
@@ -146,6 +148,14 @@
     'packages': ('openassessment',),
 }
 
+urlpatterns += (
+    url(
+        r'^openassessment/fileupload/',
+        include(orafileuploadpatterns)
+    ),
+)
```

Comment:
No, to expose the `/openassessment/fileupload` endpoint, it must be included in the base `lms.urls`.

However I can make it prettier..  see 8091ed2